### PR TITLE
Sprint test 1 - Maria

### PR DESCRIPTION
**Pull request by ptitsynamv**

### CODE REVIEW

File: Sprint_Tests/maria_ptitsyna.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+1. He used to manage well in layout and work with actual technologies in front end. After small break in several years and major depression plus listening hard music, he weakly manages in actual tendentious and drinks a lot. If he had overcome the depression after dead of his wife and child, he wouldn't have had a question about search of a job with a relevant salary.
+
```
Comment (dmitriykunev) [inline Sprint_Tests/maria_ptitsyna.md:2]: manage is used when you want to say that somebody coped with somthing in time in our case the verb BE is enough or any other variation.
He used to be really good in markup
top notch technologies
After a small break for/of several years
listening to hard music
he is not much knowlegable about modern/contemporary tendencies
after his wife's and child's death
he wouldn't have an issue with looking for a job with the relevant salary
(mixed conditional in the last sentence)

File: Sprint_Tests/maria_ptitsyna.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+2. If he had a goal and comparable wage expectation, he would already work remotely somewhere in the UAS or in Bali with a perspective to relocate in The Canary Islands and make spanish citizens. Since childhood, he has been dreaming about marriage with spanish girl and got retired early and growing up grape on a seaside of The Balearic Sea.
+
```
Comment (dmitriykunev) [inline Sprint_Tests/maria_ptitsyna.md:4]: and adequate/ reasonable salary expectation
wage is paid on a weekly basis
the perspective to relocate (we define what kind of perspective we mean)
to the Canary Islands (preposition of destination)
apply for Spanish citizenship. Language and nationality are always written with the capital letter
Spanish girl
dreaming about marriage ... and early retirement

File: Sprint_Tests/maria_ptitsyna.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+3. If you don't know how a piece of code, which was written by a colleague, works, you have you cover it by console.log and trace all logic chain. This is a time-consuming action. If we had documentation to this feature or ticket for feature realization with description of business logic, we would have flowed out the fix during the couple of hours.
+
```
Comment (dmitriykunev) [inline Sprint_Tests/maria_ptitsyna.md:6]: the piece of code (we have description what kind of code we are talking about)
you have to cover it with console.logs and trace the whole logic chain (all means plural)
or the ticket for (known info)
with the description of the business logic
I don't quite understand the sequence of actions in your conditional sentence. You suggest that documentation is about Present and Fix about the Past. Where is the logic?
a coule of hours (one couple)

File: Sprint_Tests/maria_ptitsyna.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+4. I want you to feel free to express your thoughts in foreign language in spite of
+mistakes, which can appear because of absence of practice and infrequency of language classes. I want every single one of you to read one book in the original before the end of the year, language of original is naturally English, and get pleasure from overcome a language speak barrier.
```
Comment (dmitriykunev) [inline Sprint_Tests/maria_ptitsyna.md:8]: in a foreign language
the mistakes
the absence
the language of the origin/original 
from overcoming (Gerund after prepositions)
language barrier

File: Sprint_Tests/maria_ptitsyna.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+3. If you don't know how a piece of code, which was written by a colleague, works, you have you cover it by console.log and trace all logic chain. This is a time-consuming action. If we had documentation to this feature or ticket for feature realization with description of business logic, we would have flowed out the fix during the couple of hours.
+
+4. I want you to feel free to express your thoughts in foreign language in spite of
+mistakes, which can appear because of absence of practice and infrequency of language classes. I want every single one of you to read one book in the original before the end of the year, language of original is naturally English, and get pleasure from overcome a language speak barrier. 
+
+5. On the previous week she finished the implementation of a feature and proceed to new one, for the day she has already realized much of UI. Backend had been written before she started to work on the project, that's why the swagger documentation by requests and minimal description of functionality had already been available for her. 
+
```
Comment (dmitriykunev) [inline Sprint_Tests/maria_ptitsyna.md:11]: finished .... and proceeded to a new one
for the day - is wrong we have to use SO FAR
implemented is not the same as realized
much - много most - большую часть
of the UI
documentation with requests

File: Sprint_Tests/maria_ptitsyna.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+6. We would like you to get attentively acquainted with requirements to the project and signature your agreement about nondisclosure confidential information.
```
Comment (dmitriykunev) [inline Sprint_Tests/maria_ptitsyna.md:12]: the requirements (we know what we are talking about)
and sign (verb)

File: Sprint_Tests/maria_ptitsyna.md (hunk 3 above)
Comment (ptitsynamv) [reply Sprint_Tests/maria_ptitsyna.md:6]: "I don't quite understand the sequence of actions in your conditional sentence"
About the Past - 3 condition. I guessed, we can't use "had had"
If we had had documentation ..... we would have flowed out

**Pull request merged**